install-php-extensions

```diff
@@ -3880,6 +3880,8 @@ installRemoteModule() {
 					installRemoteModule_version=2.9.8
 				elif test $PHP_MAJMIN_VERSION -le 704; then
 					installRemoteModule_version=3.1.6
+				elif test $PHP_MAJMIN_VERSION -ge 803; then
```

Comment:
Please remove these `+` (and be sure to indent lines with tabs, not spaces)

Reply:
How to hell did I miss this....

Reply:
I think I removed the `+`-sign and I'm pretty sure my IDE did tabs

Reply:
Sorry I was a bit trigger happy - I'll let you be the judge of this and let you resolve it (I saw all checks passed now).